torch/csrc/distributed/c10d/init.cpp

```diff
@@ -50,7 +50,12 @@ PyObject* c10d_init(PyObject* _unused) {
       .def(py::init<>())
       .def_readwrite("reduceOp", &::c10d::AllreduceOptions::reduceOp);
 
-  py::enum_<::c10d::ReduceOp>(module, "ReduceOp")
+  py::enum_<::c10d::ReduceOp>(module, "ReduceOp", R"(
+An enum-like class of available reduce operations: SUM, PRODUCT, MIN, MAX.
+
+The values of this class can be accessed as attributes, e.g., ``reduce_op.SUM``.
```

Comment:
s/reduce_op/ReduceOp/

Reply:
It is intentional. In python API, `reduce_op` is used instead of `ReduceOp`, so this is really adding docstring for `reduce_op`. :)

Reply:
Even for an class-like enum like this? I thought it was surfaced as `dist.ReduceOp`.

Reply:
According to our doc it is exposed as `reduce_op`. Although this might be an opportunity to rename to `ReduceOp` and deprecate `reduce_op`.

Reply:
@pietern I deperated `reduce_op` for `ReduceOp` in new commit. Let me know how it looks. :)